api/api.yaml

```diff
@@ -0,0 +1,619 @@
+openapi: 3.0.3
+info:
+  title: VCBackend
+  description: Backend component to verify credentials
+  version: 0.0.1
+tags:
+  - name: frontend
```

Comment:
We should think about extracting all frontend-exclusive stuff to either a dedicated component for demo or at least a module that can be deactivatet